airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -60,11 +66,15 @@ def get_conn(self):
             self.log.debug("loading kube_config from: default file")
             config.load_kube_config()
         else:
-            with tempfile.NamedTemporaryFile() as temp_config:
-                self.log.debug("loading kube_config from: connection kube_config")
-                temp_config.write(extras.get("extra__kubernetes__kube_config").encode())
-                temp_config.flush()
-                config.load_kube_config(temp_config.name)
+            path_or_payload = extras.get("extra__kubernetes__kube_config")
+            if not os.path.isfile(path_or_payload):
+                with tempfile.NamedTemporaryFile() as temp_config:
+                    self.log.debug("loading kube_config from: connection kube_config")
+                    temp_config.write(path_or_payload.encode())
+                    temp_config.flush()
+                    config.load_kube_config(temp_config.name)
+            else:
+                config.load_kube_config(path_or_payload)
```

Comment:
This will allow us to use multiple cllusters at the same time.
https://github.com/kubernetes-client/python-base/blob/master/config/kube_config.py#L821-L834